docs/topics/releases.md

```diff
@@ -49,6 +49,39 @@ You can also use [preview versions of Kotlin](eap.md#build-details).
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20</strong>
+            <p>Released: <strong>March 30, 2022</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>An incremental release with various improvements such as:</p>
+            <ul>
+                <li>Prototype of context receivers</li>
+                <li>Support for callable references to function interface constructors</li>
+                <li>Kotlin/Native: new memory manager</li>
+                <li>Kotlin Multiplatform: hierarchical structure support enabled by default for all new multiplatform projects</li>
+                <li>Kotlin/JS: Incremental compilation for development binaries with IR compiler</li>
+                <li>Gradle: New properties for defining Kotlin compiler execution strategy</li>
+            </ul>
+            <p>Learn more in:</p>
+            <ul>
+                <li><a href="whatsnew1620.md" target="_blank">What's new in Kotlin 1.6.20</a></li>
+                <li><a href="https://blog.jetbrains.com/kotlin/2022/03/kotlin-1-6-20-released/" target="_blank">Release blog post</a></li>
```

Comment:
@koshachy Should we mention the blog post this time? I have doubts because the blog post will differ from our previous blog posts.